This patch removes the legacy oslo namespace deprecation check from the
oslo.serialization library. The deprecated() function is left to
satisfy the unit tests. It is not suitable for the upstream and should
be removed once all of the relevant consumers have switched over to the
new namespace.

--- oslo.serialization-1.2.0/oslo/serialization/__init__.py.~1~	2015-01-08 13:32:40.000000000 -0800
+++ oslo.serialization-1.2.0/oslo/serialization/__init__.py	2015-03-02 23:09:07.429941098 -0800
@@ -21,6 +21,3 @@ def deprecated():
         DeprecationWarning,
         stacklevel=3,
     )
-
-
-deprecated()
